six/modules/c++/six.sidd/include/six/sidd/DerivedData.h

```diff
@@ -117,6 +117,11 @@ struct DerivedData: public Data
      */
     Annotations annotations;
 
+    /*
+     * (Optional) In SIDD 1.1, the LUT stored in the NITF
+     */
```

Comment:
Comment added.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Wednesday, July 13, 2016 9:04 AM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [ngageoint/six-library] Lut reposition (#86)


In six/modules/c++/six.sidd/include/six/sidd/DerivedData.h<https://github.com/ngageoint/six-library/pull/86#discussion_r70620162>:

> @@ -117,6 +117,11 @@ struct DerivedData: public Data

>       */

>      Annotations annotations;

>

> +    /*

> +     * (Optional) In SIDD 1.1, the LUT stored in the NITF

> +     */

Add a comment here about why it's not needed for SIDD 1.0

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ngageoint/six-library/pull/86/files/62bf8bc898fd0d1996e79b65c9b50004fad47da1#r70620162>, or mute the thread<https://github.com/notifications/unsubscribe/AFRAYNqJhKb7PVtj4FtrTaEDgDBw24SJks5qVOI5gaJpZM4JJsQh>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.